booking/README.md

```diff
@@ -23,16 +25,17 @@ The Booking sample app demonstrates ([browse the source](https://github.com/reve
 
 		views
 			...
+```
 
-
-
+# Database Install and Setup
+This example used [sqlite](https://www.sqlite.org/), but u can easily adopt code to your needs with mysql, postgres, et all.
```

Comment:
"et al" but "etc" is shorter. Also, "u"?? Further, why is this relevant? Isn't this implied?